admin/appearance.md

```diff
@@ -12,27 +12,25 @@ Coder offers you appearance customization options for the following:
 
 To customize your system banner messages:
 
-1. Go to **Admin > Appearance** in the Coder UI.
+1. Go to **Manage** > **Admin** > **Appearance** in the Coder UI.
```

Comment:
A little Markdown tip: if you use the 1. for all the bullets, then they will be autonumbered, which is helpful if you rearrange items. For example the following list uses 1. 1. 1.

1. List
1. Item
1. Other